protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -345,7 +345,11 @@ var (
 	{{end}}
 {{end}}
 {{if .HasQueryParam}}
-	if err := runtime.PopulateQueryParameters(&protoReq, req.URL.Query(), filter_{{.Method.Service.GetName}}_{{.Method.GetName}}_{{.Index}}); err != nil {
+	values := req.URL.Query()
+	if req.Method == http.MethodPost {
```

Comment:
I'm worried that this simple comparison will break existing POST requests using query parameters. Maybe also check that the content type is `x-www-form-urlencoded` as well?